test/adaptors/postgres-articles.test.ts

```diff
@@ -0,0 +1,144 @@
+import { literal, namedNode, quad } from '@rdfjs/data-model';
+import all from 'it-all';
+import 'jest-rdf';
+import { DatasetCore, NamedNode } from 'rdf-js';
+import pgPromise, { IBaseProtocol, IMain } from 'pg-promise';
+import PostgresArticles from '../../src/adaptors/postgres-articles';
+import ArticleNotFound from '../../src/errors/article-not-found';
+import NotAnArticle from '../../src/errors/not-an-article';
+import { schema } from '../../src/namespaces';
+import createArticle from '../create-article';
+import db from '../../src/db';
+import dataFactory from '../../src/data-factory';
+
+let pgp: IMain;
```

Comment:
Could use a clear name (thought of [something else](https://en.wikipedia.org/wiki/Pretty_Good_Privacy) on first glance!).